### PR TITLE
Deleted schema handling

### PR DESCRIPTION
**Pull request by jkleung11**
This is a cleaner version of #32 , which had changes associated with PHP 8. In an effort to minimize the changes for handling deleted schemas, we're using this PR instead. 

The `avro-serde-php` library is now using what's on master of the [storyblocks version](https://github.com/Footage-Firm/avro-serde-php). Also bumped the `events-php` version to prevent production with outdated / deleted schemas.

### CODE REVIEW

File: KafkaPhp/Serializers/AvroSerializer.php [hunk 1]
```diff
@@ -61,47 +67,61 @@ public function serialize(BaseRecord $record, string $key): array
...
-              '/Writer\'s schema .*?"name":"(\w+)".*?Reader\'s schema .*?"name":"(\w+)"/',
-              $prev->getMessage(),
-              $matches
+                '/Writer\'s schema .*?"name":"(\w+)".*?Reader\'s schema .*?"name":"(\w+)"/',
+                $prev->getMessage(),
+                $matches
             );
             [$_, $writerType, $readerType] = $matches;
 
             echo ">>> Skipping message. writerType: $writerType, readerType: $readerType" . PHP_EOL;
```
[the remark this answers is not in the document]
Comment (ilyaberd) [reply KafkaPhp/Serializers/AvroSerializer.php:100]: @jasonvideoblocks are you referring to line 84 & 101? Both don't have returns.

**Comment by ethan-schwab** [comment]
testing retrigger Circle CI

File: KafkaPhp/Serializers/AvroSerializer.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (jasonvideoblocks) [reply KafkaPhp/Serializers/AvroSerializer.php:100]: Sorry, just the `AvroDecodingException` catch block.  The block on line 101 throws an exception so it never returns anything to the caller.

[the remark this answers is not in the document]
Comment (ilyaberd) [reply KafkaPhp/Serializers/AvroSerializer.php:100]: @jasonvideoblocks Since this is not a change, we will address it later

**Pull request merged**